eshome/build.gradle

```diff
@@ -0,0 +1,86 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+
+buildscript {
+    ext {
+        pluginVersion = rootProject.properties['pluginVersion']
+        esVersion = project.properties['esVersion']
+        repoPath = rootDir.absolutePath.toString()
+    }
+}
+
+repositories {
+    mavenCentral()
+}
+
+group = 'org.elasticsearch.plugin'
+version = pluginVersion + '_es' + esVersion
+
+apply plugin: 'java'
+
+configurations {
+    runtime
+    ror
+    netty
+}
+
+dependencies {
+    ror project(path: ':es79x')
```

Comment:
I understand that we need this file to be able to run ROR with ES during development in IDEA? Could you please describe how to do it in our internal guide? It'd be nice to have sth like this: https://github.com/sscarduzio/elasticsearch-readonlyrest-plugin/wiki/Development#run-es-as-a-java-application (I'll move the old guide some day)

Reply:
done